shared/src/business/entities/cases/Case.ts

```diff
@@ -1910,7 +1916,7 @@ export class Case extends JoiValidationEntity {
    * @returns {Case} this case entity
    */
   fileCorrespondence(correspondenceEntity) {
-    this.correspondence = [...this.correspondence, correspondenceEntity];
+    this.correspondence = [...this.correspondence!, correspondenceEntity];
```

Comment:
Is this correct?

Perhaps we should remove the `?` on the property of the Case Class that suggests it's optional?